Dockerfile

```diff
@@ -133,55 +258,73 @@ RUN /opt/ort/bin/import_proxy_certs.sh && \
         tar -zxC . && \
         (cd CocoaPods-9461b346aeb8cba6df71fd4e71661688138ec21b && \
             gem build cocoapods.gemspec && \
             gem install cocoapods-1.10.1.gem \
         ) && \
-        rm -rf CocoaPods-9461b346aeb8cba6df71fd4e71661688138ec21b
+        rm -rf CocoaPods-9461b346aeb8cba6df71fd4e71661688138ec21b && \
+    # Update installed bundler
+    gem install bundler  -v '~> 2.2.0' && \
+    gem install json -v '~> 2.5.0'
+#    # Add cocoa pods
+#    git clone https://github.com/CocoaPods/CocoaPods.git && \
+#        cd CocoaPods && \
+#        git checkout $COCOAPODS_TAG && \
+#        gem build cocoapods.gemspec && gem install *.gem && cd .. && \
+#        pod repo add main https://github.com/CocoaPods/Specs.git --allow-root
 
-# Add scanners (in versions known to work).
-ARG SCANCODE_VERSION
-RUN curl -ksSL https://github.com/nexB/scancode-toolkit/archive/v$SCANCODE_VERSION.tar.gz | \
-        tar -zxC /usr/local && \
-        # Trigger ScanCode configuration for Python 3 and reindex licenses initially.
-        cd /usr/local/scancode-toolkit-$SCANCODE_VERSION && \
-        PYTHON_EXE=/usr/bin/python3 /usr/local/scancode-toolkit-$SCANCODE_VERSION/scancode --reindex-licenses && \
-        chmod -R o=u /usr/local/scancode-toolkit-$SCANCODE_VERSION && \
-        ln -s /usr/local/scancode-toolkit-$SCANCODE_VERSION/scancode /usr/local/bin/scancode
+# Pull in Rust from build image
+RUN mkdir -p /opt/rust
+COPY --from=rust-build /opt/rust /opt/rust
+RUN ln -s /opt/rust/cargo/bin/cargo /usr/local/bin
+
+# Pull in go and godep from build image
+RUN mkdir -p /opt/go
+COPY --from=go-build /opt/go /opt/go
+RUN ln -s /opt/go/bin/go /usr/local/bin
 
-COPY --from=build /usr/local/src/ort/cli/build/distributions/ort-*.tar /opt/ort.tar
+# Pull in flutter from build image
+RUN mkdir -p /opt/flutter
+COPY --from=flutter-build /opt/flutter /opt/flutter
+RUN ln -s /opt/flutter/bin/cache/dart-sdk/bin/pub /opt/flutter/bin/cache/dart-sdk/bin/dart*  /usr/local/bin
+
+# Pull in scancode from build image
+RUN mkdir -p /opt/scancode
+COPY --from=scancode-build /opt/scancode /opt/scancode
+RUN ln -s /opt/scancode/scancode-toolkit-${SCANCODE_VERSION}/scancode /usr/local/bin/scancode
+
+# Pull in Android SDK from build image
+RUN mkdir -p ${ANDROID_HOME}
+COPY --from=android-sdk-buiild ${ANDROID_HOME} ${ANDROID_HOME}
+
+# Pull in ORT from build image
+COPY --from=java-build /usr/local/src/ort/cli/build/distributions/ort-*.tar /opt/ort.tar
 
 RUN tar xf /opt/ort.tar -C /opt/ort --strip-components 1 && \
-    rm /opt/ort.tar && \
-    /opt/ort/bin/ort requirements
+    rm /opt/ort.tar
+    # Disabled to due hardcoded scancode version in requirements check
+    # /opt/ort/bin/ort requirements
+
+COPY --from=java-build /usr/local/src/ort/helper-cli/build/scripts/orth /opt/ort/bin/
+COPY --from=java-build /usr/local/src/ort/helper-cli/build/libs/helper-cli-*.jar /opt/ort/lib/
+
+COPY curations.tar /
```

Comment:
`ADD` can combine a `COPY` and extract: https://docs.docker.com/engine/reference/builder/#add

Reply:
I tried building this container, but it fails on this COPY as the curations.tar isn't available (anymore).